wordcloud_test.py

```diff
@@ -2,19 +2,18 @@
 from wordcloud import main
 from visual_wordcloud import VisualWordCloud
 
-TEXT = """This is a test test string string string."""
+# TEXT = "This is a test test string string string."
+TEXT = "initial_text.txt"
```

Comment:
Is TEXT supposed to be a filename?

If you want to load your text from a file you could use the function you already created.
Inside test_word_cloud() you could do:

```
    text = read_file(FILENAME)
    word_to_count = main(text, "alphabetical")
```
Where FILENAME = "initial_text.txt"
Note that you'll need to import read_file() from your wordcloud module.

Reply:
Yes, thank you for the description.  I realise now that the word_cloud.py file can be used as a series of discrete functions.  I had assumed that there was one entry point for arguments (main()), and then one exit point for return statements (return word_to_count), so had never looked at it any other way.  Hopefully, therefore, I’m now managing it more appropriately.

As per above, the read_file() function is now accessed directly from the wordcloud_test program.

Reply:
Brilliant. You've got it 👍 

word_cloud.py can contain functions that we import and use in other modules. It can also contain classes. 😄